### PR TITLE
{perf}[foss/2020b] Extrae v3.8.0

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/e/Extrae/Extrae-3.8.0-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+description = """Extrae is the core instrumentation package developed by
+the Performance Tools group at BSC. Extrae is capable of instrumenting
+applications based on MPI, OpenMP, pthreads, CUDA1, OpenCL1, and StarSs1
+using different instrumentation approaches. The information gathered by
+Extrae typically includes timestamped events of runtime calls,
+performance counters and source code references. Besides, Extrae
+provides its own API to allow the user to manually instrument his or her
+application."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/e/Extrae/Extrae-3.8.0-foss-2020b.eb:17]: @robert-mijakovic can this be moved down to `gompi`?

Comment (robert-mijakovic) [reply easybuild/easyconfigs/e/Extrae/Extrae-3.8.0-foss-2020b.eb]: @migueldiascosta Moved! Thanks for suggesting it. I just based it on the previously existing easyconfig and didn't think about it.

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @robert-mijakovic!

**Pull request merged**